### PR TITLE
Fix/214

### PR DESCRIPTION
**Pull request by Epiq99**
Made DigitTextField use Textfield from material-ui.
DigitTextField is now also stateless.

### CODE REVIEW

**Comment by Epiq99** [comment]
oops, i did not see the other files change as well

File: src/elements/digit-text-area/DigitTextArea.element.jsx [hunk 1]
```diff
@@ -137,7 +71,7 @@ DigitTextArea.propTypes = {
     error: PropTypes.bool,
     /** If error is true, then this errorMessage will be shown instead of lowerLabel */
     errorMessage: PropTypes.string,
-    /** If true, then you can't edit this text area. */
+    /** If true, then you can"t edit this text area. */
```
Comment (Hoidi) [inline src/elements/digit-text-area/DigitTextArea.element.jsx]: Undo this

Comment (Epiq99) [reply src/elements/digit-text-area/DigitTextArea.element.jsx]: Done

**Comment by Portals** [comment]
This was already "stateless" before since it didn't use a state. However, you turned it into a function component which is awesome!

**Pull request merged**